src/groupeddataframe/splitapplycombine.jl

```diff
@@ -1064,6 +1113,10 @@ function _combine(f::AbstractVector{<:Pair},
         else
             if source_cols isa Int
                 incols = (parentdf[!, source_cols],)
+            elseif source_cols isa AsTable
+                incols = Tables.columntable(select(parentdf,
```

Comment:
Why not use the indexing syntax (here and elsewhere), for consistency with other branches?

Reply:
A good question :). The reason is that `source_cols.colselector` can be a `ColumnIndex`, in which case indexing syntax would not produce a table, but a vector, and then `Tables.columntable` would be broken. (I will make sure that we test for this)